app/src/main/java/id/rizmaulana/covid19/data/mapper/CovidDataMapper.kt

```diff
@@ -41,3 +43,21 @@ object CovidPinnedDataMapper {
         response.lastUpdate)
     else null
 }
+
+object CovidDetailDataMapper {
+
+    fun transform(responses: List<CovidDetail>?, @CaseTypes caseType: Int) = responses?.map { response ->
+        LocationItem(
+            response.confirmed,
+            response.recovered,
+            response.deaths,
+            response.locationName,
+            response.lastUpdate,
+            response.lat,
+            response.long,
+            response.countryRegion,
+            response.provinceState,
+            caseType
+        )
+    }.orEmpty().sortedBy { it.locationName }
```

Comment:
any reason why you change the order of the list?

Reply:
I thought it will be nice to sort it alphabetically, should I use the order from the API instead?

Reply:
it is arguable, as we don't have the data let's put it as default from API, and user might be more interested in numbers which comes from the default

Reply:
Ohh I see, okayyy thanks for the input